### PR TITLE
TeamMLG.net

### PR DESCRIPTION
**Pull request by plushTeddy**
## Type of change

- [ ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/teammlg/manifest.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+		},
+		"cwbwtraining": {
+			"name": "CwBwTraining",
+			"color": "#23bade",
+			"versions": "1.8.8<*"
+		}
+	},
+	"discord": {
+		"server_id": 935962104475820083,
+		"rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/teammlg/manifest.json:58]: Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

File: minecraft_servers/teammlg/manifest.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	"direct_ip": "teammlg.net",
+	"server_wildcards": [
+		"%.teammlg.net"
+	],
+	"supported_languages": [
+		"de",
+		"en"
+	],
+	"social": {
+		"web": "https://teammlg.net",
```
Comment (JakobBruening) [inline minecraft_servers/teammlg/manifest.json:13]: Website not available.

File: minecraft_servers/teammlg/manifest.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		"web_shop": "https://store.teammlg.net",
```
Comment (JakobBruening) [inline minecraft_servers/teammlg/manifest.json:14]: Website not available.

File: minecraft_servers/teammlg/manifest.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+		"web_support": "https://teammlg.net/forum",
```
Comment (JakobBruening) [inline minecraft_servers/teammlg/manifest.json:15]: Website not available.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
- Compress your image(s)
- Fix fomatting of your `manifest.json`, spacing is completely wrong

File: minecraft_servers/teammlg/manifest.json (hunk 4 above)
Comment (plushTeddy) [reply minecraft_servers/teammlg/manifest.json:15]: Could you please push it? The website will go online in 1-2 weeks!

**Comment by JakobBruening** [comment]
Still:
- Compress your image(s)
- Fix fomatting of your manifest.json, spacing is completely wrong

File: minecraft_servers/teammlg/manifest.json (hunk 4 above)
Comment (JakobBruening) [reply minecraft_servers/teammlg/manifest.json:15]: No, please remove. You can create a new PR THEN. Usually, we push PRs fast, so there is no reason to already show a dead link.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

**Comment by JakobBruening** [comment]
Please re-open as soon as changes are done.

**Pull request closed without merging**